common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -180,8 +180,8 @@
                     this.loadDiscussions(this.$el, function() {
                         self.hideDiscussion();
                         DiscussionUtil.discussionAlert(
-                            gettext('Sorry'),
-                            gettext('We had some trouble loading the discussion. Please try again.')
+                            gettext('Error'),
+                            gettext('This discussion could not be opened. Refresh the page to try again.')
```

Comment:
@andy-armstrong I was trying to avoid "loaded", but I see we use it elsewhere. Sorry for the extra revision, but let's go back to "This discussion could not be loaded. Refresh the page to try again."